src/renderer/components/cluster-manager/cluster-status.tsx

```diff
@@ -150,6 +150,10 @@ export class ClusterStatus extends React.Component<Props> {
   }
 
   render() {
+    if (!this.entity) {
+      return null;
+    }
```

Comment:
Extra checks is good but it doesn't related to pod logs? And is there a possibility now to see empty cluster screen with one more reason? 

If we check for `entity` so it fixes some app crash issue?

Reply:
Ah oops, I meant to move this into a separate PR.